tests/test_utils/test_timer.py

```diff
@@ -0,0 +1,46 @@
+# tests.test_utils.test_timer
+# Tests for the stand alone timer functions in Yellowbrick utils.
+#
+# Author:   ZJ Poh <poh.zijie@gmail.com>
+# Created:  Tue Jul 17 21:11:11 2018 -0700
+#
+# Copyright (C) 2017 District Data Labs
+# For license information, see LICENSE.txt
+"""
+Tests for the stand alone timer functions in Yellowbrick utils.
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import unittest
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
+from yellowbrick.utils.timer import *
+
+##########################################################################
+## Helper Function Tests
+##########################################################################
+
+class TestTimer(unittest.TestCase):
```

Comment:
We're trying to move over to [pytest](https://docs.pytest.org/en/latest/) and though we haven't fully deprecated our use of `unittest`, it would help us when we do, if we could have these be pytest tests by default. To do this, would you mind changing the definition of the class to `TimerTests(object)`? The tests below will then also require `assert` statements, which I'll point out below.

Reply:
Sure, I will move over the pytest.